recipes/tdastro/conda_build_config.yaml

```diff
@@ -0,0 +1,2 @@
+python_min:
```

Comment:
While this file works. I recommend to remove it and add a line at the top of the recipe with:

```
{% set python_min = "3.10" %}
```

That way you only need to edit a single file when updating.